packages/suite/src/hooks/wallet/useCardanoStaking.ts

```diff
@@ -62,7 +64,10 @@ export const useCardanoStaking = (): CardanoStaking => {
         throw Error('useCardanoStaking used for other network');
     }
 
+    const alreadyVoted = account.misc?.staking?.drep !== null;
     const device = useSelector(selectSelectedDevice);
+    const cardanoNetwork = account.symbol === 'ada' ? 'mainnet' : 'preview';
+    const { trezorDRep } = useSelector(state => state.wallet.cardanoStaking[cardanoNetwork]);
```

Comment:
maybe line 70 + 69 (https://github.com/trezor/trezor-suite/pull/16003/files#r1899652157) could be a selector